.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,11 @@
 
 <!-- Describe your changes in detail. Why is this change required? What problem does it solve? -->
 
+### 🤖 AI-Assisted Contributions
```

Comment:
i think we can omit the heading

Reply:
I've removed this heading and incorporated all the changes. 😊